### PR TITLE
Improve doc formatting and add api update instructions

### PR DESCRIPTION
**Pull request by friscoMad**
## Description
Added sub header for common errors, as it was all mixed and not easy to read.
Fixed formula formatting
Added FAQ about api updating.
Added error about certificates in RedHat based distros

## Motivation and Context
Help the world

## How Has This Been Tested?
Not tested but MD valid (can we test RtD commits?)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/common-issues/faq.md [hunk 1]
```diff
@@ -97,8 +97,20 @@ No, remove everything and start from scratch. A Pull Request is merged when it m
 
 Good Job! We recommend making new accounts. [Current Tools are Here!](https://rocketmap.readthedocs.io/en/develop/extras/Community-Tools.html)
 
+### I am not sure how to update api
+
+You will get an error when you try to start your server with an different api that the one it is build for, to update your api to the required version do:
+
```
Comment (jagauthier) [inline docs/common-issues/faq.md]: "built" for.

File: docs/common-issues/faq.md [hunk 2]
```diff
@@ -97,8 +97,20 @@ No, remove everything and start from scratch. A Pull Request is merged when it m
 
 Good Job! We recommend making new accounts. [Current Tools are Here!](https://rocketmap.readthedocs.io/en/develop/extras/Community-Tools.html)
 
+### I am not sure how to update api
+
+You will get an error when you try to start your server with an different api that the one it is built for, to update your api to the required version do:
+
```
Comment (BrownSlaughter) [inline docs/common-issues/faq.md]: clarify that the scanner will be paused if using an older api that niantic currently enforces

> You will get an error that the scanner is paused when you try to start your server with an older api that the one niantic currently enforces, to update your api to the required version do:

File: docs/common-issues/faq.md [hunk 3]
```diff
@@ -97,8 +97,20 @@ No, remove everything and start from scratch. A Pull Request is merged when it m
 
 Good Job! We recommend making new accounts. [Current Tools are Here!](https://rocketmap.readthedocs.io/en/develop/extras/Community-Tools.html)
 
+### I am not sure how to update api
```
Comment (sebastienvercammen) [inline docs/common-issues/faq.md]: `api` > `pgoapi`

File: docs/common-issues/faq.md (hunk 2 above)
Comment (sebastienvercammen) [inline docs/common-issues/faq.md]: ^ and capitalize `API`.

File: docs/common-issues/faq.md [hunk 4]
```diff
@@ -129,51 +145,79 @@ KeyError: 'map_cells'
...
 
-st=step distance  
-sd=scan delay [default: 10]  
-w=# of workers  
+st=step distance\
+sd=scan delay [default: 10]\
+w=# of workers\
 t=desired scan time  
 
 ### Speed Scan
```
Comment (sebastienvercammen) [inline docs/common-issues/faq.md]: `Hex scan` below doesn't have `scan` capitalized, so for consistency we should change it.

File: docs/common-issues/faq.md [hunk 5]
```diff
@@ -129,51 +145,79 @@ KeyError: 'map_cells'
...
 
-Workers for initial scan(speed scan):  
-Workers = Cells / 20, Cells = (((steps * (steps - 1)) * 3) + 1)  
+Workers for initial scan(speed scan):
+```
+Cells = (((steps * (steps - 1)) * 3) + 1)  
+Workers = Cells / 20
```
Comment (sebastienvercammen) [inline docs/common-issues/faq.md]: In formulas, don't capitalize the variable names, and don't mix upper- and lowercase (`Cells/Workers` vs `steps`). Let's just go for `workers, cells, steps`.

**Comment by friscoMad** [comment]
Fixes #2091

File: docs/common-issues/faq.md [hunk 6]
```diff
@@ -129,51 +147,79 @@ KeyError: 'map_cells'
...
+```
+
+RedHat based distros (Fedora, CentOS) could have an old OpenSSL version that is not compatible to the latest `certifi` package, to fix it you need to:
+
+```bash
+pip uninstall certifi
+pip install certifi==2015.4.28
+```
+
+User `sudo` or `--user` if you are not using an account with root permission.
```
Comment (sebastienvercammen) [inline docs/common-issues/faq.md]: I believe you meant to type "use" instead of "user" at the start.

**Pull request merged**